src/Pages/SearchResult/SearchResult.js

```diff
@@ -10,69 +35,57 @@ class SearchResult extends Component {
-                </div>
-              </div>
-            </div>
-
-            <div className="buttonContainer">
-              <div className="buttonBox">
-                <div className="locationIcon"></div>
-                <button className="button">지도</button>
-              </div>
-              <div className="buttonBox">
-                <div className="filterIcon"></div>
-                <button className="button">필터</button>
+              {OPTIONS.map((el) => {
+                return (
+                  <div className="optionBox" key={el.id}>
+                    <span className="optionType">{el.type}</span>
+                    <div className="optionWrapper">
+                      <span className="option">{el.option}</span>
+                      <div className={el.icon} />
+                    </div>
+                  </div>
+                );
+              })}
+              <div className="buttonContainer">
+                {BUTTON.map((el) => {
+                  return (
+                    <div className="buttonBox" key={el.id}>
+                      <div className={el.icon} />
+                      <button className="button">{el.name}</button>
+                    </div>
+                  );
+                })}
               </div>
             </div>
           </div>
-        </div>
-
-        <div className="categoryBarContainer">
-          <div className="categoryBox">
-            <ul className="categoryList">
-              <li>전체 </li>
-              <div className="categoryBar"></div>
-              <li>시간단위</li>
-              <div className="categoryBar"></div>
-              <li>패키지단위</li>
-              <div className="categoryBar"></div>
-              <li>월단위</li>
-            </ul>
-
-            <div className="checkBoxContainer">
-              <input className="checkingDiscount" type="checkbox" />
-              <span className="discountSpace">지금 쿠폰 할인되는 공간</span>
-              <span className="newTag">NEW</span>
+          <div className="categoryBarContainer">
+            <div className="categoryBox">
+              <ul className="categoryList">
+                {CATEGORY.map((el) => {
+                  return (
+                    <>
+                      <li>{el.list}</li>
+                      <div className={el.bar ? "categoryBar" : ""} />
+                    </>
+                  );
+                })}
+              </ul>
+              <div className="checkBoxContainer">
+                <input className="checkingDiscount" type="checkbox" />
+                <span className="discountSpace">지금 쿠폰 할인되는 공간</span>
+                <span className="newTag">NEW</span>
+              </div>
             </div>
+            <select>
+              {SELECT.map((el) => {
+                return (
+                  <>
+                    <option value={el.value}>{el.order}</option>
+                  </>
+                );
+              })}
```

Comment:
return 요소가 한 개 밖에 없어서 그냥 option 태그만 작성해도 좋겠네요. arrow function 언제 중괄호, return 키워드 생략 가능한지 정리해주세요.
```suggestion
              {SELECT.map( el => <option value={el.value}>{el.order}</option>)}
```